ui/imports/shared/popups/CommunityIntroDialog.qml

```diff
@@ -0,0 +1,86 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.1
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+import StatusQ.Popups 0.1
+
+StatusModal {
+    id: root
+
+    property string name
+    property string introMessage
+    property url imageSrc
+
+    signal joined
+
+    // Code below is needed because StatusModal content padding is messed up
+    // FIXME: when StatusModal is reworked
+    width: undefined       // popup is able to determine size from its content
+    topPadding: 64 + 16    // 64 is header height
+    leftPadding: 16
+    rightPadding: 16
+    bottomPadding: 71 + 16 // 71 is footer height
```

Comment:
Looks like hack :) what a reason for it?

Reply:
The answer is just few lines above :)

https://github.com/status-im/status-desktop/blob/db661806d0925381b3a81c4d721e7fea9c0d48ff/ui/imports/shared/popups/CommunityIntroDialog.qml#L20